src/app/item-card/item-card.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="item">
+  <div class="__container">
+    <img class="__item-image" [src]="itemData.image" />
```

Comment:
Is there some error with class names?

The main one should be ItemCard, as is the name of our component.
And then ItemCard__image (object) or ItemCard--big (modifyer).

We follow OOCSS and BEM -> https://github.com/airbnb/css#oocss-and-bem